core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -221,6 +221,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorNavbarBreadcrumbDirective.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorNavigationDirective.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationPropertyServices.js"></script>
```

Comment:
verify the file name.
ExplorationPropertyService.js not ExplorationPropertyServices.js (`s` in the end).
Reason behind E2E test faliures.